transifex_announcement

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env ruby
+
+require 'date'
+
+if ARGV.length != 3
+  puts "Usage: #{$0} <ga_date> <schedule_url> <planning_url>"
+  puts "Example: VERSION=3.16 #{$0} 2025-09-09 'https://community.theforeman.org/t/foreman-3-16-schedule-and-planning/43345' 'https://community.theforeman.org/t/string-freeze-and-state-of-translations-foreman-3-16/44042'"
+  exit 1
+end
+
+version = ENV['VERSION']
+if version.nil? || version.empty?
+  puts "Error: VERSION environment variable must be set"
+  exit 1
+end
+
+ga_date = ARGV[0]
+schedule_url = ARGV[1]
+planning_url = ARGV[2]
+
+# Validate date format
+begin
+  Date.parse(ga_date)
+rescue ArgumentError
+  puts "Error: Invalid date format. Please use YYYY-MM-DD format."
+  exit 1
+end
+
+user_name = `git config --global user.name`.strip
+
+announcement = <<~EOF
```

Comment:
This contains a lot of empty lines. Is that intentional?

Reply:
It was intentional, IIRC. It was to make the paragraphs in the announcement message in Transifex visually better separated from each other. Let's remove the empty lines for now. If that announcement won't look good, I'll make a quick patch to add them once after I "officially" use the script for the first time.